Modules/_sqlite/module.c

```diff
@@ -702,6 +735,11 @@ module_exec(PyObject *module)
         goto error;
     }
 
+    /* Set the keyword tuple */
```

Comment:
This comment does not add any value (nor does the other similar comments above it, but that's not for this PR). I'd prefer if we do not add comments that spells out in text what the code clearly does.

Reply:
Addressed with 370dd8b1dba22f499ae8bc4f0bc5f881c948cfd4